UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/ChemicalMetabolismSystem.cs

```diff
@@ -145,13 +145,12 @@ public void MetaboliseReactions()
 				foreach (var metabolismComponent in PrecalculatedMetabolismReactions[Reaction]) //TODO maybe lag? Alternative?
 				{
 					ProcessingAmount += metabolismComponent.ReagentMetabolism * metabolismComponent.BloodThroughput *
-					                    metabolismComponent.CurrentBloodSaturation *
-					                    Mathf.Max(0.10f, metabolismComponent.RelatedPart.TotalModified);
+					                    metabolismComponent.CurrentBloodSaturation;
 				}
 
 				if (ProcessingAmount == 0) continue;
 
-				//Reaction.React(PrecalculatedMetabolismReactions[Reaction], _reagentPoolSystem.BloodPool, ProcessingAmount);
```

Comment:
https://user-images.githubusercontent.com/34368774/230813874-0c442cca-92d5-446c-b142-05059c97f1d8.mp4

Reply:
yeah, it was commented out when the system was implemented but not turned on xD